universal/router/index.js

```diff
@@ -0,0 +1,21 @@
+/* @flow */
+import React from 'react'
+
+import LayoutCore from 'layouts/LayoutCore'
+
+import match from './matches'
+import routes from './routes'
+
+const Router = ({ store }: { store: Object }) => {
+  return (
+    <LayoutCore>
+      <div>
```

Comment:
good catch....it was at one point complaining about wrapping a single child...see [this issue](https://github.com/ReactTraining/react-router/issues/4131) if you want context...but since I wrapped it with the Layout, it's the same result as having a <div>.

Thanks!